CRM/Core/BAO/SchemaHandler.php

```diff
@@ -666,6 +664,23 @@ public static function safeRemoveFK($table_name, $constraint_name) {
     $dao = CRM_Core_DAO::executeQuery($query, $params);
 
     if ($dao->fetch()) {
+      $found = TRUE;
+    }
+    return $found;
+  }
+
+  /**
+   * Remove a foreign key from a table if it exists.
+   *
+   * @param $table_name
+   * @param $constraint_name
+   *
+   * @return bool
+   */
+  public static function safeRemoveFK($table_name, $constraint_name) {
+    $check = self::checkFKExists($table_name, $constraint_name);
+
+    if ($check) {
```

Comment:
stylistically I would put the fn call inside the IF rather than use a throw away var